src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, Prop, h, Event, EventEmitter } from '@stencil/core';
+import {
+    MenuItem,
+    LimelMenuCustomEvent,
+    ListSeparator,
+    ActionBarItem,
+    OpenDirection,
+} from '@limetech/lime-elements';
+
+/**
+ * @private
+ */
+@Component({
+    tag: 'limel-action-bar-overflow-menu',
+    shadow: false,
+})
+export class ActionBarOverflowMenu {
+    /**
+     * List of the items that should be rendered in the overflow menu.
+     */
+    @Prop()
+    items: Array<MenuItem | ListSeparator>;
+
+    /**
+     * Defines the location that the content of the overflow menu
+     * appears, in relation to its trigger.
+     */
+    @Prop({ reflect: true })
+    public openDirection: OpenDirection;
+
+    @Event()
+    public select: EventEmitter<ActionBarItem>;
+
+    public render() {
+        return [
+            <limel-menu
```

Comment:
We need to add an accessible label for this (for screen readers). For example: "See 4 additional actions". Or at least this needs a prop so that the consumer can add it.

Reply:
I actually change my mind about this. The button is already like this: `<button>+11</button>`. It should be accessible enough I guess.
<img width="459" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/e4ce8846-9e02-42f5-bce4-596bc3a5db7f">